Signal/src/AppDelegate.m

```diff
@@ -368,6 +419,11 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 {
     OWSAssertIsOnMainThread();
 
+    if (self.didAppLaunchFail) {
+        OWSFail(@"%@ %s", self.logTag, __PRETTY_FUNCTION__);
```

Comment:
Can we include a substring like "app launch failed" in all of these log/failure strings?

Reply:
Done.